runner/execer/os/os_execer.go

```diff
@@ -70,7 +71,7 @@ func (e *osExecer) Exec(command execer.Command) (result execer.Process, err erro
 		return nil, err
 	}
 
-	proc := &osProcess{cmd: cmd}
+	proc := &osProcess{cmd: cmd, LogTags: tags.LogTags{RequestorTag: command.RequestorTag}}
```

Comment:
If you used a LogTags field instead of embedding it in Command, you could just do "LogTags: command.LogTags". (I'd try to differentiate the naming a bit more between the LogTags type and the LogTags fields).

Either way, we should make sure to keep the LogTags entries together where possible rather than split it out like here.